### PR TITLE
Test: e2e tests for features

### PR DESCRIPTION
**Pull request by Ajen07**
# Description

- e2e test for `features controller`

## Issue link

[Fixes # (issue)](https://app.clickup.com/t/86azr0q4j)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Feature updates / changes
- [x] Tests
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- tested with  `yarn test:docker`

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have updated the change log

### CODE REVIEW

File: src/features/features.controller.ts [hunk 1]
```diff
@@ -196,7 +207,7 @@ export class FeaturesController {
         } else {
             throw new HttpException(
                 "user is unauthorized to perform this action",
```
Comment (cherylli) [inline src/features/features.controller.ts:209]:
```suggestion
                "Access denied: You do not have sufficient permissions to perform this action",
```

**Review by cherylli — DISMISSED** [review]
everything looks fine, just one minor suggestion so the error response matches the forbidden error

Also just a note all forbidden stuff will need to be moved to our CASL authorization, but shouldn't affect the tests. In fact it's good to have these tests now, so we know we don't mess up when updating it

**Review by JoshuaHinman — APPROVED** [review]
All tests passed - looks good

File: src/features/features.controller.ts (hunk 1 above)
Comment (Ajen07) [reply src/features/features.controller.ts]: Currently in some controllers there are no permissions for own_team , after the permissions are added then, we can add the forbidden tests to it.

**Pull request merged**